Cargo.toml

```
@@ -0,0 +1,2 @@
+[workspace]
+members = ["generator", "proxmox-api"]
```

```suggestion
members = ["generator", "proxmox-api"]
resolver = "2"
```

There's some warning about which resolver version to use, please add this